<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>VlanWidget</class>
 <widget class="QWidget" name="VlanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>193</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Parent interface:</string>
     </property>
     <property name="buddy">
      <cstring>parent</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="parent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>VLAN &amp;id:</string>
     </property>
     <property name="buddy">
      <cstring>id</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="id">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>VLAN inter&amp;face name:</string>
     </property>
     <property name="buddy">
      <cstring>ifaceName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KLineEdit" name="ifaceName"/>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="reorderHeaders">
     <property name="text">
      <string>Output packet &amp;headers reordering</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="gvrp">
     <property name="toolTip">
      <string>GARP VLAN Registration Protocol</string>
     </property>
     <property name="text">
      <string>&amp;GVRP</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="looseBinding">
     <property name="text">
      <string>&amp;Loose binding</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>parent</tabstop>
  <tabstop>id</tabstop>
  <tabstop>ifaceName</tabstop>
  <tabstop>reorderHeaders</tabstop>
  <tabstop>gvrp</tabstop>
  <tabstop>looseBinding</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
